data/hai/hai ships.txt

```diff
@@ -1053,7 +1053,7 @@ ship "Shield Beetle"
 	sprite "ship/hai shield beetle"
 	thumbnail "thumbnail/hai shield beetle"
 	attributes
-		category "Heavy Warship"
+		category "Medium Warship"
```

Comment:
What's the reasoning for this?

Reply:
I made this change before the mass rebalance, when the shield beetle was considerably faster. Heavy warship probably is more appropriate now.

Reply:
```suggestion
		category "Heavy Warship"
```